src/main/java/com/commercetools/sync/taxcategories/utils/TaxCategoryUpdateActionUtils.java

```diff
@@ -80,17 +79,7 @@ private TaxCategoryUpdateActionUtils() {
         @Nonnull final TaxCategoryDraft newTaxCategory,
         @Nonnull final TaxCategorySyncOptions syncOptions) {
```

Comment:
When you update a code you need to update the other parts, please remove unused syncOptions from the method,  also in tests, Additionally please check buildRemoveOrReplaceTaxRateUpdateActions it should not return an exception in the method contract, so you need an update for those.

Reply:
ok would remove syncOptions from argument in UpdateActionUtil class

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/524/commits/f14e6e60730a4b579100cd4a2e253cbd146ed869